students/minha/westagram/users/views.py

```diff
@@ -63,21 +63,22 @@ def post(self, request):
             return JsonResponse({'message':'KEY_ERROR'}, status=400)
 
 
-class LogInView(View):
+class SignInView(View):
     def post(self, request):
         try:
-            data            = json.loads(request.body)
+            data         = json.loads(request.body)
+
+            user         = User.objects.get(email=data['email'])
+            access_token = jwt.encode({"id" : user.id}, settings.SECRET_KEY, algorithm = settings.ALGORITHM)
 
-            email_log_in    = data['email']
-            password_log_in = data['password']
+            if not bcrypt.checkpw(data['password'].encode('utf-8'), user.password.encode('utf-8')):
+                return JsonResponse({"message" : "INVALID_PASSWORD"}, status=401)
```

Comment:
아이디가 잘못 되었는지, 비밀번호가 잘못 되었는지 모르게 메시지를 보내야 합니다.
```suggestion
                return JsonResponse({"message" : "INVALID_USER"}, status=401)
```